File: adress-service/src/main/java/br/com/adressservice/core/utils/GraphqlSchemaReaderUtil.java
package br.com.adressservice.core.utils;

import lombok.extern.slf4j.Slf4j;

import java.io.IOException;

@Slf4j
public final class GraphqlSchemaReaderUtil {

    public static String getSchemaFromFileName(final String filename) throws IOException {
        return new String(
                GraphqlSchemaReaderUtil.class.getClassLoader().getResourceAsStream("graphql/" + filename + ".graphql").readAllBytes());
    }
}

File: validated-document-service/src/main/kotlin/br/com/validateddocumentservice/core/utils/GraphqlSchemaReaderUtil.kt
package br.com.validateddocumentservice.core.utils

import java.io.IOException


object GraphqlSchemaReaderUtil {
    @Throws(IOException::class)
    fun getSchemaFromFileName(filename: String): String {
        return String(
            GraphqlSchemaReaderUtil::class.java.classLoader.getResourceAsStream("graphql/$filename.graphql")!!
                .readAllBytes()
        )
    }
}
